app/code/core/Mage/Adminhtml/controllers/CacheController.php

```diff
@@ -63,7 +67,11 @@ public function flushAllAction()
      */
     public function flushSystemAction()
     {
+        Mage::app()->getCacheInstance()->banUse('config');
+        Mage::getConfig()->reinit();
         Mage::app()->cleanCache();
+        Mage::app()->getCacheInstance()->unbanUse('config');
+        Mage::getConfig()->saveCache();
         Mage::dispatchEvent('adminhtml_cache_flush_system');
         $this->_getSession()->addSuccess(Mage::helper('adminhtml')->__("The OpenMage cache storage has been flushed."));
         $this->_redirect('*/*');
```

Comment:
I've put a `die()` here, I was expecting to find something in the var/cache folder but it's empty

```
fab@MBP14-Fab ~/P/openmage (save-after-flush)> ls var/cache/ -lah
Permissions Size User Date Modified Name
drwxrwxrwx     - fab  20 Sep 21:35  .
drwxr-xr-x     - fab   1 Aug 08:56  ..
```

am I missing something?

Reply:
Oops, I copied the method and forgot to change the false to true... Pushed a fix and confirmed the config cache is saved.